ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -206,6 +217,73 @@ def _agent_evp_proxy_is_available(self):
                 return True
         return False
 
+    @classmethod
+    def test_skipping_enabled(cls):
+        if not cls.enabled:
+            return False
+        return getattr(cls._instance, "_test_skipping_enabled_by_api", False)
```

Comment:
nit: I think this is a bit cleaner than using `getattr`
```suggestion
        return cls._instance and cls._instance._test_skipping_enabled_by_api
```